daily-programming/2024-06-14/main0614.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+  int n;
+  cin >> n;
+  int l[n];
+  int max = 0, others = 0;
+  for(int i = 0; i < n; i++) {
+    cin >> l[i];
+    if(l[i] > max) {
+      others += max;
+      max = l[i];
+    } else {
+      others += l[i];
+    }
+  }
+  if(max < others) {

```

Comment:
標準入力を受け取ってすぐ使ってしまうのであれば、配列は使わなくても解けそうです。
```suggestion
  int edge;
  int max_edge = 0, edges_length = 0;
  for(int i = 0; i < n; i++) {
    cin >> edge;
    if(edge > max_edge) {
      edges_length += max_edge;
      max_edge = edge;
    } else {
      edges_length += edge;
    }
  }
  if(max_edge < edges_length) {
```